### ISSUE TITLE
Add Graphql Types

### ISSUE THREAD

**Issue by gdvalderrama** [yourlabs/django-cities-light#279]
**This is a feature request.**

Consider adding Graphql types for the [models](https://github.com/yourlabs/django-cities-light/blob/master/src/cities_light/abstract_models.py).

Other apps, like django countries, offer graphql types for their models, [example](https://github.com/SmileyChris/django-countries/blob/main/django_countries/graphql/types.py).

The type is a minimal representation to avoid imposing on the user.
Having these types saves time when using the app with graphql.

An example would look something like:

```
from graphene import ObjectType, String


class CountryType(ObjectType):

    name = String(description="Country name.")
    name_ascii = String(description="Country name ascii.")
    slug = String(description="Country slug.")
    geoname_id = Int(description="Country geoname id.")
    alternate_names = String(description="Country alternate names.")
    code2 = String(description="Country code 2 letters.")
    code3 = String(description="Country code 3 letters.")
    continent = String(description="Country continent.")
    tld = String(description="Country top level domain.")
    phone = String(description="Country phone code.")
```

**Comment by marianoeramirez** [comment]
It would be great if you can provide some feedback here or test in your projects. 

https://github.com/yourlabs/django-cities-light/pull/281

**Comment by gdvalderrama** [comment]
> It would be great if you can provide some feedback here or test in your projects.
> 
> #281

The types look good, but the tests added are failing:

> relation "cities_light_country" does not exist

**Comment by marianoeramirez** [comment]
Released on V3.10.0
